src/index.html

```diff
@@ -22,6 +22,408 @@
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="#"
+          />
+
+          <div class="card__description card__description--margin--top">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code card__code--margin--top">
+            Product code: 195434
+          </div>
+          <div class="card__rating card__rating--margin--top">
+            <ul class="stars">
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+            </ul>
+
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__priceBlock card__priceBlock--margin--top">
+            <div class="card__txt">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+
+          <div data-qa="card-hover">
+            <a
+              class="card__button card__button--margin--top"
+              href="#"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="#"
+          />
+
+          <div class="card__description card__description--margin--top">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code card__code--margin--top">
+            Product code: 195434
+          </div>
+          <div class="card__rating card__rating--margin--top">
+            <ul class="stars">
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+              <li class="stars__area stars__area--size"></li>
+            </ul>
+
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__priceBlock card__priceBlock--margin--top">
+            <div class="card__txt">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+
+          <div data-qa="hover">
+            <a
+              class="card__button card__button--margin--top"
+              href="#"
```

Comment:
The 'href' attribute is set to '#' for the 'BUY' button. This will not lead to any specific page or action. Consider setting this to a valid URL or implementing JavaScript to handle the click event.